spikeinterface/sorters/kilosortbase.py

```diff
@@ -49,3 +141,8 @@ def _get_result_from_folder(cls, output_folder):
         keep_good_only = sorter_params.get('keep_good_only', False)
         sorting = KiloSortSortingExtractor(folder_path=output_folder, keep_good_only=keep_good_only)
         return sorting
+
+    @classmethod
+    def _get_specific_options(cls):
+        # need to be implemented in subclass
+        raise NotImplementedError
```

Comment:
@chyumin shouldn't this just append some entries to the `opt` dictionary? The base version could be something like this?
```
@classmethod
def _get_specific_options(cls, ops, params):
     return ops
```

what do you think?